src/components/Theme/src/Theme.vue

```diff
@@ -54,35 +44,47 @@ export default {
 		resolveTheme(data, this.parentTheme, this.theme, this.profile);
 		return data;
 	},
-	computed: {
-		styles() {
-			const { colors } = this;
-
-			return {
-				'--neutral-0': colors['neutral-0'],
-				'--neutral-10': colors['neutral-10'],
-				'--neutral-20': colors['neutral-20'],
-				'--neutral-80': colors['neutral-80'],
-				'--neutral-90': colors['neutral-90'],
-				'--neutral-100': colors['neutral-100'],
-				'--color-background': colors.background,
-				'--color-heading': colors.heading,
-				'--color-text': colors.text,
-				'--color-elevation': colors['color-elevation'],
-				'--color-overlay': colors['color-overlay'],
-			};
-		},
-	},
 	beforeUpdate() {
-		// update theme on prop changes
 		resolveTheme(this.$data, this.parentTheme, this.theme, this.profile);
+		this.applyTheme();
+	},
+	beforeMount() {
+		this.applyTheme();
+	},
+	methods: {
+		applyTheme() {
+			const { theme } = createStitches({ theme: this.theme, prefix: 'maker' });
+			this.defaultTheme = theme;
```

Comment:
i don't think this line is doing anything?

Reply:
Yeah, you're right. I pulled this over from an older implementation. Originally, I was passing this as the the provide data to all the components, but in this iteration it's not necessary. I'll remove it in this PR.

This would set us up to completely remove the default-theme.js. The default styles would be set either globally in a css file or respectively in each component. The theme injection would then provide the ability for external consumers to pass that data.